### PR TITLE
sync some fixes from the server

### PR DESCRIPTION
**Pull request by elianiva**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,8 @@ RUN npm install
 
 ENV NODE_ENV=production
 
-RUN npm run build \
+RUN npm install \
+    && npm run build \
```
Comment (aldy505) [inline Dockerfile:12]: di line 7 masih ada `npm install`, diapus aja

File: src/index.ts [hunk 2]
```diff
@@ -49,6 +49,7 @@ const eventHandlers: EventHandlerMapping = {
 
 // webhook server and handlers
 const polkaInstance = polka();
+polkaInstance.get("/", (_, res) => void res.writeHead(200).end("OK"));
```
Comment (aldy505) [inline src/index.ts:52]: so dirty lol

**Review by aldy505 — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 2 above)
Comment (elianiva) [reply src/index.ts:52]: aneh lihat `=> { res.writeHead(...) }`

**Comment by aldy505** [comment]
wkwkwk okay no prob

**Pull request merged**